### DISCUSSION TITLE
It's "check out" and not "checkout"

### DISCUSSION THREAD

**Discussion by wristshot0** [community/community#37110] [Other Feature Feedback, Questions, & Ideas]
You don't "checkout" anything. Like "log in" is to "login," "check out" is the verb form for "checkout," which is a noun. GitHub, please change this and stop looking stupid. You can leave the command alone obviously, but in plain text on the site, it's "check out."

<img width="485" alt="Screen Shot 2022-10-24 at 8 49 55 AM" src="https://user-images.githubusercontent.com/14323199/197529223-f2017915-eb37-479d-86f8-cdb93cd19281.png">

**Comment by DenverCoder1** [comment]
~This has nothing to do with GitHub, as the term "checkout" actually goes all the way back to Git.~

~https://git-scm.com/docs/git-checkout~

~Note that this is a CLI command meaning every word in the command is a separate argument to the command. There would never be a space in an argument since that would make it into two separate arguments. You do occasionally see hyphenated arguments, for example, in the case of [`git cherry-pick`](https://git-scm.com/docs/git-cherry-pick) but usually that is only done for readability. In nearly all CLI applications there is a convention to not separate words in argument names, therefore `login` would be much more expected than `log in` and `checkout` more than `check out`.~

~There is also evidence of "checkout" being turned into a Git term as a word without spaces, as you can see that they have opted to not separate the words even when another hyphenated word is added on (eg. [`git checkout-index`](https://git-scm.com/docs/git-checkout-index))~

~Also, by the way [checkout](https://en.wiktionary.org/wiki/checkout) is an English word on its own.~

----

After rereading your question it seems I misunderstood what you were asking.

As you are pointing out, "checkout" is a noun from the verb phrase "check out", so when written in plain text, it would make more sense to use the verb phrase.

I think it would be fine for GitHub to change it, as even the [Git documentation](https://git-scm.com/docs/git-checkout) uses "check out" when describing the operation. The way GitHub has it is also correct, though, as the "checkout" operation is commonly known to Git users so when it says "Checkout" in "Checkout with GitHub CLI" it really means "Run the checkout operation".

**Reply by wristshot0** [reply]
Yes--I'm not arguing with anything technical here (though check-out might make more sense given you are "checking out," I'm not going to die on that sword). The issue is a pet peeve of mine: in the language on the site. "Run the checkout operation" would be OK too to your point.
